semanticdb/metacp/src/main/scala/scala/meta/metacp/Settings.scala

```diff
@@ -1,21 +1,20 @@
 package scala.meta.metacp
 
-import io.github.soc.directories.ProjectDirectories
 import scala.meta.cli._
+import scala.meta.internal.io.PathIO
 import scala.meta.io.AbsolutePath
 import scala.meta.io.Classpath
-import scala.meta.internal.metacp.BuildInfo
 
 final class Settings private (
-    val cacheDir: AbsolutePath,
+    val out: AbsolutePath,
     val classpath: Classpath,
```

Comment:
Given what we discussed earlier today, how about changing `classpath: Classpath` to `in: AbsolutePath`?

Reply:
I vote for "no" because

- we use `Classpath` in `ConvertSuite`
- running `Classpath.entries.map(Metacp.process)` has a big performance penalty because `ClasspathIndex` needs to be re-created every time

I would prefer to come up with a scheme so that metacp produces one jar per classpath entry.

Reply:
Alright, let's have metacp produce one jar per classpath entry.